dev-util/jenkins-bin/jenkins-bin-2.20.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit user systemd
+
+DESCRIPTION="Extensible continuous integration server"
+HOMEPAGE="http://jenkins-ci.org/"
+LICENSE="MIT"
+SRC_URI="http://mirrors.jenkins-ci.org/war/${PV}/${PN/-bin/}.war -> ${P}.war"
+RESTRICT="mirror"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE=""
+
+RDEPEND="media-fonts/dejavu
+	media-libs/freetype
+	!dev-util/jenkins-bin:lts
+	>=virtual/jre-1.7.0"
```

Comment:
Yes. We need to make sure there's indeed a JVM installed and that's what the `java-pkg-2` eclass is there for.